virt/virt-2-6-release-notes.adoc

```diff
@@ -60,6 +60,7 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 
 //CNV-9456: When importing VM disk images, you can now control which multus network is used
 
+* The CDI can now xref:../virt/virtual_machines/virtual_disks/virt-using-preallocation-for-datavolumes.adoc#virt-using-preallocation-for-datavolumes[preallocate disk space] to improve write performance of importing and uploading operations and when creating blank data volumes.
```

Comment:
This sentence could be rewritten for clarity. Maybe something like "The Containerized Data Importer (CDI) can now preallocate disk space to import and upload data at a faster speed and when creating blank data volumes."